packages/ui-toolkit/src/components/atoms/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -1,48 +1,44 @@
 import React from 'react';
 
 import './toggleSwitch.css';
+import '@groww-tech/mint-css/dist/index.css';
 
 
 const ToggleSwitch = (props: Props) => {
   const {
-    switchCircleColor, activeBackgroundColor, width,
-    inactiveBackgroundColor, onChange, height,
+    onChange, isDisabled,
```

Comment:
line break props, each one into a new line. Increases readability.

Reply:
To be done when props are more than 4

Reply:
Got it.